docs(readme): added description new structure and onboarding new product teams and special cases (#2)
catenax-ng/product-onboarding: PREREQUISITES.md

# Requirements for Internal System Team Members

Our current environments are running in Azure Cloud based on Azure Kubernetes Service and were managed or deployed with Terraform scripts. The following Tools are required to run theese scripts.

## Azure CLI

Using our Azure Subscription you have to be invited to our Subscription and installed Azure CLI.
Links to our Subsrciption ()

- [Azure CLI Installing Guidelines](https://learn.microsoft.com/en-us/cli/azure)

> You had to be invited to our Azure Subscription on [Azure-Portal](https://portal.azure.com/#@catenax.onmicrosoft.com)

Reviewer: ```suggestion
You have to be invited to our Azure Subscription on [Azure-Portal](https://portal.azure.com/#@catenax.onmicrosoft.com)
```

> Login to Azure on CLI for help, just ask an System Team Internal Member who has allready done that.

Reviewer: ```suggestion
Login to Azure on CLI for help, just ask a System Team Internal Member who has already done that.
```

``` bash
az login --tenant <Catena-X-TenantID>
az account set --subscription <SubscriptionID>
## Get Kubeconfig Details for earch Kubernetes Cluster
az aks get-credentials --resource-group <RessourceGroup-of-desired-Cluster> --name <AzureName-of-Kubernetes-Cluster>
## Testing which kubenetes contexts you allready have
kubectl config get-contexts        
```

## Terraform

- [Terraform Installing Guidelines](https://developer.hashicorp.com/terraform/downloads)

``` bash
# Installation for terraform autocomplete for autocompletion of terraform cmd's
terraform -install-autocomplete
# Reload your Terminal / CLI  based on your Working OS
# Testing if autocompletion is working install / init should appeare
terraform i
# Switch to a terraform file-directory for e.g our github-folder and initialize terraform
terraform init
```

- enable your CLI

> Depending on which terraform run you will about to plan and perform you need to export the desired `TOKEN` from the terraform run to generate the ressources within it.

Reviewer: I'd remove this for easier understanding if the next sentence is modified.

```suggestion
```

> For `terraform apply` and `terraform plan` command the following command line variables has to be set:

Reviewer: ```suggestion
For `terraform apply` and `terraform plan` command the following command line variables has to be exported from our `Vault` and set:
```
